Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM node:16-stretch AS development
+FROM node:16-alpine AS development
+
+RUN apk --no-cache add --virtual .builds-deps build-base python3 git
```

Comment:
I'm curious why stretch needs to be replaced with alpine. Is this related to the ARM issues you had?

Reply:
Actually, it doesn't have to necessarily. The most important line is
```
RUN apk --no-cache add --virtual .builds-deps build-base python3 git
```

With stretch, I would need to change this to the `apt` equivalent I guess. I prefer `alpine` though because the image is way smaller (~ 5 MB) in contrast to stretch (~ 40 MB). Or is there any specific reason to use the `stretch` variant?